hooks/k8s-bootstrap/yaml/namespace.yaml

```diff
@@ -5,3 +5,5 @@ metadata:
   name: "$NAMESPACE"
   labels:
     app.kubernetes.io/name: open-ondemand
+  annotations:
+    last-hook-execution: "$TIMESTAMP"
```

Comment:
I'm sure this is a part of the other repo, but shouldn't all this stuff be namespaced like `openondemand.org/last-hook-execution` or `osc.edu/last-hook-execution`?

Reply:
Ya, I changed to be namespaced to `openondemand.org/`.  If this was in like osc-ood-configs I'd use `osc.edu/`.